examples/freesurfer_segs/freesurfer_provenance.jsonld

```diff
@@ -0,0 +1,53 @@
+{
+	"@context": [
+		"https://purl.org/nidash/bidsprov/context.json",
+		{
+			"freesurfer": "https://surfer.nmr.mgh.harvard.edu/",
+			"description": "http://purl.org/dc/terms/description"
+		}
+	],
+	"BIDSProvVersion": "1.0.0",
+	"records": {
+		"prov:Agent": [
+			{
+				"@id": "RRID:SCR_001847",
+				"@type": "prov:SoftwareAgent",
+				"label": "Freesurfer",
+				"version": "6.0.0"
+			}
+		],
+		"prov:Activity": [
+			{
+				"@id": "niiri:ff2754fe-1585-11eb-b1e7-1094bbf2086c",
+				"description": "Freesurfer segmentation statistics",
+				"startedAtTime": "2018-11-27T21:42:49",
+				"wasAttributedTo": "RRID:SCR_001847",
+				"freesurfer:ColorTable": "/opt/freesurfer-6.0.0-min/ASegStatsLUT.txt",
+				"freesurfer:ColorTableTimeStamp": "2017/01/18 22:00:02",
+				"freesurfer:ExcludeSegId": "0 2 3 41 42",
+				"freesurfer:Excluding": "Cortical Gray and White Matter",
+				"freesurfer:InVolFile": "mri/norm.mgz",
+				"freesurfer:InVolFileTimeStamp": "2018/11/27 21:42:49",
+				"freesurfer:InVolFrame": "0",
+				"freesurfer:NRows": "45",
+				"freesurfer:NTableCols": "10",
+				"freesurfer:PVVolFile": "mri/norm.mgz",
+				"freesurfer:PVVolFileTimeStamp": "2018/11/27 21:42:49",
+				"freesurfer:SUBJECTS_DIR": "/working/derivatives/mindboggle/freesurfer_subjects",
+				"freesurfer:SegVolFile": "mri/aseg.mgz",
+				"freesurfer:SegVolFileTimeStamp": "2018/11/28 04:51:14",
+				"freesurfer:Title": "Segmentation Statistics",
+				"freesurfer:VoxelVolume_mm3": "1",
+				"freesurfer:anatomy_type": "volume",
+				"freesurfer:cmdline": "mri_segstats --seg mri/aseg.mgz --sum stats/aseg.stats --pv mri/norm.mgz --empty --brainmask mri/brainmask.mgz --brain-vol-from-seg --excludeid 0 --excl-ctxgmwm --supratent --subcortgray --in mri/norm.mgz --in-intensity-name norm --in-intensity-units MR --etiv --surf-wm-vol --surf-ctx-vol --totalgray --euler --ctab /opt/freesurfer-6.0.0-min/ASegStatsLUT.txt --subject sub-0050660",
```

Comment:
A `command` field has be added (as REQUIRED) in the definition of a Activity from our [gdoc](https://docs.google.com/document/d/1vw3VNDof5cecv2PkFp7Lw_pNUTUo8-m8V4SIdtGJVKs/edit?ts=60cb5cd9#)

So I guess `freesurfer:cmdline` should be renamed `command` and placed above (eg. after `startedAtTime`)

Reply:
Got it, will make the change....